src/allotropy/parsers/bmg_labtech_smart_control/constants.py

```diff
@@ -0,0 +1,13 @@
+from allotropy.allotrope.models.shared.components.plate_reader import SampleRoleType
+
+DISPLAY_NAME = "BMG SMART Control"
```

Comment:
```suggestion
DISPLAY_NAME = "BMG Labtech SMART Control"
```
Sorry just noticed this

Reply:
(will need to update auto-generated tables as well)